### DISCUSSION TITLE
Ruleset templates

### DISCUSSION THREAD

**Discussion by BeritJanssen** [Amsterdam-Music-Lab/MUSCLE#760] [Ideas]
Perhaps instead of *one* ruleset template, we could have multiple ones, for common experiment types, e.g.,
- [x] preference (no assumptions about playlist)
- [ ] 2 alternative forced choice (playlist will have to encode some information in `group` or `tag` of sections)
- [ ] showcase different players
- [ ] hooked (potentially separate templates for "SongSync" and "SongBool" rounds)
- [ ] matching pairs
- [ ] staircasing

**Comment by drikusroor** [comment]
If it's much work to add them all at once, we could also opt for adding them incrementally based on request and utility.
